samples/artifacts/DataPipelining/ConnectSiddhiApps/ConnectSiddhiAppsSource.siddhi

```diff
@@ -0,0 +1,30 @@
+@App:name("ConnectSiddhiAppsSource")
+@App:description("Connecting Siddhi apps using In-Memory transport and view the output on the console.")
+
+/*
+
+Purpose:
+    This application demonstrates how to configure a Siddhi application to connect multiple Siddhi Apps deployed under the same Siddhi Manager and log the events in OutputStream to the output console. The application consumes events which are published by "ConnectSiddhiAppsSink" app.
+
+Prerequisites:
+    1) Save this sample
+
+Executing the Sample:
+    1) Start the Siddhi application by clicking on 'Run'.
+    2) If the Siddhi application starts successfully, the following messages would be shown on the console.
+        * ConnectSiddhiAppsSource.siddhi - Started Successfully!
+
+Note:
+    Both "ConnectSiddhiAppsSink" app and "ConnectSiddhiAppsSource" app should be running at the same time in order to view the results on the console.
```

Comment:
Move note as executing sample step 1 and say that the other siddhi app has intuctions

Reply:
The requested changes are addressed by the commit 6d13673.